e2e/rbd.go

```diff
@@ -1515,6 +1551,43 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+
+			By("create a PVC-PVC clone with object-map,fast-diff image-features and bind it to an app", func() {
+				// pvc clone is only supported from v1.16+
+				if !k8sVersionGreaterEquals(f.ClientSet, 1, 16) {
+					Skip("pvc clone is only supported from v1.16+")
```

Comment:
it seems that these `Skip()` calls cause skipping of all the following `By()` tests as well. Instead log a warning and return from the `By()` function.

Ceph-CSI does not try to support Kubernetes < 1.16 anymore, so this check can be remove completely.